lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -2586,6 +2582,39 @@ lldb::addr_t SwiftLanguageRuntime::GetAsyncContext(RegisterContext *regctx) {
   return LLDB_INVALID_ADDRESS;
 }
 
+// Creates an expression accessing (fp - 8), with an optional dereference
+// operation. This is only valid for x86_64 or aarch64.
+llvm::ArrayRef<uint8_t>
+getAsyncRegFromFramePointerDWARFExpr(llvm::Triple::ArchType triple,
```

Comment:
lldb functions start with an Upper case letter.

Reply:
Fixed in the new version